### DISCUSSION TITLE
프로젝트 이름 === 어플 이름 추천받습니다!

### DISCUSSION THREAD

**Discussion by Sleepingoff** [FRONTENDSCHOOL7/VILLAINS#3] [Q&A] [answered]
## 프로젝트 내용
1호선을 기준으로 지하철 시간표 조회 및 실시간 위치 정보 등을 활용하여 사용자가 작성한 게시글을 지도에 표시해 주고, 지하철 택배, 간단한 채팅 등으로 커뮤니티의 성격을 가지고 있습니다.

**Comment by Sleepingoff** [comment]
1호선에서 만나

**Comment by Sleepingoff** [comment]
villains on map

**Reply by Sleepingoff** [reply]
지도 위의 빌런들인데, 지도에 게시글을 표시해주는 기능에서 착안하였습니다.

**Comment by kys0417** [comment]
1단타(일단타)

일단 타 와 1호선 1 합친 말장난입니다 저희가 지하철 정보 주니까 일단 1호선 타라는.. 의미입니다..

**Comment by gyulls2** [comment]
빌런떴다

커뮤니티에만 집중되는 느낌이지만 아이디어 내봅니다. .. ..

**Comment by Sleepingoff** [comment]
su1bway

**Comment by gyulls2** [comment]
지하철에서 만나

다른 호선까지 다 커버됩니다 ,, . ...

**Comment by CreateSong** [comment, marked as answer]
후보1. SUB ONE (서브원): subway + one 합성어
후보2. 위기탈출 넘버원: 1호선 빌런이 많다고 들었습니다..위기탈출 넘버원...ㅠ
조심스럽게,,추천 남기고 떠나보겠습니다,,화이팅!!!

**Reply by Sleepingoff** [reply]
서브원 감사합니다~!
